space-plugins/nuxt-base/modules/tunnel.ts

```diff
@@ -0,0 +1,16 @@
+import { defineNuxtModule } from '@nuxt/kit';
+import { execaCommand } from 'execa';
+
+export default defineNuxtModule({
+	setup(_options, nuxt) {
+		const tunnelCommand = process.env.TUNNEL_COMMAND;
+		if (process.env.NODE_ENV !== 'development' || !tunnelCommand) {
```

Comment:
Hey @eunjae-lee this might not be ready yet, I am just leaving comments as I am OOO tomorrow, but not to block you. 

nit: `process.env.NODE_ENV !== 'development'` is this check needed? From what I understood, the `listen` hook will only be triggered when dev server is loaded. But I might be missing something here. 

question: What happens when the dev server is closed, will the tunnel still be running?

Reply:
I actually added the explanation in the PR description, but if the dev server is closed, the tunnel will be closed as well, because the tunnel is the child process of the dev server process. When the parent dies, the child dies as well. (horrific dev story)

`process.env.NODE_ENV !== 'development'` -> I guess we won't really need it because `listen` is for the dev server as you said, however, if we have this, the production build will not include the code within the if clause at all, and it means we won't include `execa` dependency as well in the bundle output. I actually referred to this https://github.com/nuxtus/localtunnel/blob/main/src/module.ts#L44

Reply:
😂😂 

Totally missed the description sorry. I read it now, thank you for the details. Everything looks perfect. Can't wait to try it 🫰🏻